src/Features/Core/Portable/InitializeParameter/AbstractAddParameterCheckCodeRefactoringProvider.cs

```diff
@@ -537,13 +537,25 @@ private static TStatementSyntax CreateNullCheckStatement(SemanticModel semanticM
         }
 
         private static SyntaxNode CreateArgumentException(
-            Compilation compilation, SyntaxGenerator generator, IParameterSymbol parameter)
+            Compilation compilation, SyntaxGenerator generator, IParameterSymbol parameter, string methodName)
         {
-            // Note "message" is not localized.  It is the name of the first parameter of 
-            // "ArgumentException"
+            string text = methodName switch
+            {
+                nameof(string.IsNullOrEmpty) => new LocalizableResourceString(nameof(FeaturesResources.cannot_be_null_or_empty), FeaturesResources.ResourceManager, typeof(FeaturesResources)).ToString(),
+                nameof(string.IsNullOrWhiteSpace) => new LocalizableResourceString(nameof(FeaturesResources.cannot_be_null_or_whitespace), FeaturesResources.ResourceManager, typeof(FeaturesResources)).ToString(),
+                _ => "default"
+            };
+            var content = new List<SyntaxNode>()
+            {
+                generator.InterpolatedStringText(generator.InterpolatedStringTextToken($"'{{{generator.NameOfExpression(generator.IdentifierName(parameter.Name))}}}'")),
+                generator.InterpolatedStringText(generator.InterpolatedStringTextToken(text))
+            };
             return generator.ObjectCreationExpression(
                 GetTypeNode(compilation, generator, typeof(ArgumentException)),
-                generator.LiteralExpression("message"),
+                generator.InterpolatedStringExpression(
+                    generator.CreateInterpolatedStringStartToken(false).WithLeadingTrivia(content.First().GetLeadingTrivia()),
```

Comment:
named arg for `false`

Reply:
the first/last content nodes won't have any trivia.  so i don't know why w're taking trivia from them and adding to the interpolated string.